fix_distance_to_turns.py

```
@@ -0,0 +1,81 @@
+# `let map = world.get_resource::<Map>();`
+# But if it's too complex to sample the map, we can just say "average cost is 1.0" or use a straight line sample.
+# "移動コストを加味した" -> let's take the straight line path and sum the movement cost!
+# Or just get the movement cost of `Plains` (平地) for the unit's `movement_type`.
+# Let's see how we can write this neatly.
+
+replacement = """
+            let stats = unit_registry.get_stats(ut).unwrap();
+            let max_movement = std::cmp::max(1, stats.max_movement) as isize;
+            let max_range = stats.max_range as isize;
+
+            // 平均的な移動コストをマスターデータ（例えば平地）から取得するか、簡易的に1とする
```

Ruff (0.15.10)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)